xlive/H2MOD_Halo2Final.cpp

```diff
@@ -16,7 +17,8 @@
 */
 
 bool isEnabled = false;
-bool isHosting = false;
+bool H2Server = (h2mod->Server) ? 1 : 0;
```

Comment:
not sure having a bool really helps readability in this case, 

you're just making a new bool of an existing bool.